anvio/migrations/modules/v1_to_v2.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+import sys
+import argparse
+import re
+
+import anvio.db as db
+import anvio.tables as t
+import anvio.utils as utils
+import anvio.terminal as terminal
+
+from anvio.errors import ConfigError
+
+current_version, next_version = [x[1:] for x in __name__.split('_to_')]
+
+run = terminal.Run()
+progress = terminal.Progress()
+
+
+def migrate(db_path):
+    if db_path is None:
+        raise ConfigError("No database path is given.")
+
+    utils.is_modules_db(db_path)
+
+    # make sure the current version is 1
+    modules_db = db.DB(db_path, None, ignore_version = True)
+    if str(modules_db.get_version()) != current_version:
+        modules_db.disconnect()
+        raise ConfigError("Version of this modules database is not %s (hence, "
+                          "this script cannot really do anything)." % current_version)
+
+    # split any orthology entries with multiple KOs into separate lines
+    where_clause = "data_name = 'ORTHOLOGY' AND (data_value LIKE '%+%' OR data_value LIKE '%-%')"
+    bad_rows = modules_db.get_some_rows_from_table(t.module_table_name, where_clause)
+
+    modules_db.remove_some_rows_from_table(t.module_table_name, where_clause)
+
+    good_rows = []
+    for mod, orth, kos, definition, line in bad_rows:
+        if orth != "ORTHOLOGY":
+            raise ConfigError("Serious SQL error here. We asked for ORTHOLOGY but we got %s" % orth)
+        split_kos = re.split('\+|\-', kos)
+        for ko in split_kos:
+            if len(ko) != 6 or ko[0] != 'K':
+                raise ConfigError("Uh oh. We split a KO that doesn't seem to have the right format. It looks like this: %s, "
+                                  "and the string that it came from is %s." % (ko, kos))
+            good_rows.append((mod, orth, ko, definition, line))
+
+    modules_db.insert_many(t.module_table_name, entries=good_rows)
```

Comment:
Ah, sorry! I've missed this totally.

Migration scripts should never use table descriptions in the tables module. The reason to this is this: imagine someone run this script a year from now, and `t.module_table_name` in `anvio.tables` is different in that anvi'o version than what it is now. This will crash.

You should always use explicit names, and ideally never import `anvio.tables`.

Reply:
No worries! I will fix this :)